pkg/build/graph/analysis/bc.go

```diff
@@ -12,16 +12,19 @@ import (
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildedges "github.com/openshift/origin/pkg/build/graph"
 	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
+	imageapi "github.com/openshift/origin/pkg/image/api"
 	imageedges "github.com/openshift/origin/pkg/image/graph"
 	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
 const (
-	TagNotAvailableWarning     = "ImageStreamTagNotAvailable"
-	LatestBuildFailedErr       = "LatestBuildFailed"
-	MissingRequiredRegistryErr = "MissingRequiredRegistry"
-	MissingImageStreamErr      = "MissingImageStream"
-	CyclicBuildConfigWarning   = "CyclicBuildConfig"
+	TagNotAvailableWarning       = "ImageStreamTagNotAvailable"
+	LatestBuildFailedErr         = "LatestBuildFailed"
+	MissingRequiredRegistryErr   = "MissingRequiredRegistry"
+	MissingImageStreamErr        = "MissingImageStream"
+	CyclicBuildConfigWarning     = "CyclicBuildConfig"
+	MissingInputImageStreamError = "MissingInputImageStreamError"
+	MissingImageStreamTagWarning = "MissingImageStreamTagWarning"
```

Comment:
I read `TagNotAvailableWarning` based on `FindPendingTags` as referring to an output image tag.

Are you amenable to renaming `TagNotAvailableWarning` to `MissingOutputImageStreamWarning` ?